modules/acquia_cms_headless/acquia_cms_headless.install

```diff
@@ -33,3 +35,16 @@ function acquia_cms_headless_requirements($phase) {
 
   return $requirements;
 }
+
+/**
+ * Install and update new headless roles.
+ */
+function acquia_cms_headless_update_8001() {
+  \Drupal::configFactory()->getEditable('user.role.headless')->set('label', 'Headless Administrator')->save();
```

Comment:
@deepakmishra2 Let's not use Drupal config factory to change label. Use the similar as below i.e
```suggestion
  $role = Role::load('headless');
  if ($role instanceOf Role) {
    //write code to update role label, permissions etc.
  }
```
Same as below update hook as well.

Reply:
Also, there are 3 more permissions added to headless role. See [this](https://backlog.acquia.com/browse/ACMS-1316) ticket.